common/util.go

```diff
@@ -382,3 +382,15 @@ func DoWithOverrideReadOnlyOnAzureFiles(ctx context.Context, action func() (inte
 	_, err = action()
 	return err
 }
+
+// @brief Thia API check if the container name provided is a system container or not
+func IsSystemContainer(containerName string) bool {
```

Comment:
is it possible that the input $ is encoded when it gets here? Would be good to check that case as well